README.md

```diff
@@ -27,122 +25,76 @@ iOS targets iOS 10.0+. Aries Bifold can only be run on physical devices as of ri
 
 ### Testing
 
-Aries Mobile Agent React Native aims to utilize the [Aries Protocols Test Suite (APTS)](https://github.com/hyperledger/aries-protocol-test-suite) and potentially the [Aries Agent Test Harness (AATH)](https://github.com/hyperledger/aries-agent-test-harness) to test for Aries Agent compatibility and interoperability.
-
-## Install
-
-
-You can also watch a recording of setting up and running the mobile wallet and receiving a credential using the ACA-Py demo. Watch the video [here](https://youtu.be/AomoHvw4lgc) (thanks [@xtrycatchx](https://github.com/xtrycatchx)).
-
-### Prerequistes
-
-#### React Native
+Aries Bifold utilizes the [Aries Agent Test Harness (AATH)](https://github.com/hyperledger/aries-agent-test-harness) to test for Aries Agent compatibility and interoperability.
 
-Installation instructions are documented [here](https://reactnative.dev/docs/environment-setup).
+Aries Bifold also utilizes the [Aries Toolbox](https://github.com/hyperledger/aries-toolbox) & [aries-acapy-plugin-toolbox](https://github.com/hyperledger/aries-acapy-plugin-toolbox) for issuer and verifier roles.
 
-Troubleshooting guides can be found in the [docs/installation](./docs/INSTALLATION.md) directory.
+## Install
```

Comment:
This section contains guides on installation, configuration and getting the agent running. I think a more common and familiar structure to most devs is having individual `install`, `configure` and `run` sections.